include/swift/Frontend/Types.def

```diff
@@ -48,6 +48,7 @@ TYPE("dependencies",    Dependencies,       "d",               "")
 TYPE("autolink",        AutolinkFile,       "autolink",        "")
 TYPE("swiftmodule",     SwiftModuleFile,    "swiftmodule",     "")
 TYPE("swiftdoc",        SwiftModuleDocFile, "swiftdoc",        "")
+TYPE("swiftinterface",  SwiftModuleInterfaceFile, "swiftinterface", "")
```

Comment:
Is it worthwhile adjusting the spacing on the other lines to maintain the columnar alignment?

Reply:
Yeah, I'm not sure why I was hesitant to do so. Will fix.